test/integration/features/basic.feature

```diff
@@ -84,6 +84,14 @@ Feature: Basic test
         Then stderr should contain "Checking if user is a member of the Hyper-V Administrators group"
         Then stderr should contain "Checking if the Hyper-V virtual switch exist"
 
+    @darwin @linux @windows
+    Scenario: Request start with monitoring stack
+        When setting config property "enable-openshift-monitoring" to value "true" succeeds
```

Comment:
@jsliacan This is changed to `enable-cluster-monitoring` so it need to be updated.